### ISSUE TITLE
Increase efficency of predict in TFT

### ISSUE THREAD

**Issue by nicocheh** [jdb78/pytorch-forecasting#1047]
I have a TFT model with this parameters and quantile loss. When i run predict in dataset of batches of 100.000 series of 16 time points each one (i.e a dataset of 1.600.000 rows), it gets 6 minutes to predict that. Is there any way I can make this more efficent? Is the predict method using the GPU? Is it paralellized? Thank you very much @jdb78 

          learning_rate=0.0992,
          hidden_size=100,  # most important hyperparameter apart from learning rate
          # number of attention heads. Set to up to 4 for large datasets
          attention_head_size=4,
          dropout=0.283,  # between 0.1 and 0.3 are good values
          hidden_continuous_size=89,  # set to <= hidden_size
          output_size=len(quantiles),  # 7 quantiles by default
